dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -521,7 +582,15 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
             case (input, tx) => input.toFundingInput(tx.transaction)
           })
 
-          updatedDLCDb <- updateDLCContractIds(offer, accept)
+          dlcDb <- updateDLCContractIds(offer, accept)
+
+          builder = DLCTxBuilder(offer, accept.withoutSigs)
+          fundingTx <- builder.buildFundingTx
+          outPoint = TransactionOutPoint(fundingTx.txId, UInt32.zero)
```

Comment:
just curious, why is it safe to use outpiont index `0`?

Reply:
Currently, the funding output is always at that index